src/gridstack.js

```diff
@@ -41,9 +41,10 @@
         },
 
         sort: function(nodes, dir, width) {
-            width = width || _.chain(nodes).map(function(node) { return node.x + node.width; }).max().value();
+            width = width || _.chain(nodes).map(function(node) {
+                return parseInt(node.x) + parseInt(node.width);}).max().value();
             dir = dir != -1 ? 1 : -1;
-            return _.sortBy(nodes, function(n) { return dir * (n.x + n.y * width); });
+            return _.sortBy(nodes, function(n) { return dir * (parseInt(n.x) + parseInt(n.y) * width); });
```

Comment:
For compatibility across environments you should pass the radix like so: `parseInt(n.x, 10)`. See better explanation/background here https://eslint.org/docs/rules/radix